### ISSUE TITLE
Valet has problems with executing commands from another drive.

### ISSUE THREAD

**Issue by luukhoeben** [cretueusebiu/valet-windows#12]
Hey,

Everytime I try to run a valet command from my D drive in git bash, it says:
```Could not start the phpfpmservice service. Check ~/.valet/Log for errors.```

I am running it as administrator, even in the default windows command prompt as administrator it doesn't work for some reason. When I try to run a command from my C drive it works, the problem is that I have a small boot drive (120GB SSD) so I can't really store much on there.

Is there any way to fix this issue?

Greetings,
Luuk Hoeben

**Comment by cretueusebiu** [comment]
Are you on the latest version ? 
Do you have Valet installed in the D drive or just your project.

**Comment by luukhoeben** [comment]
I'm on 2.0.8 and I have installed it using composer on my C drive, my parked folder is in the D drive.

**Comment by cretueusebiu** [comment]
It doesn't matter. But can you start Valet from the C driver and then just park/link sites from the D drive ?

**Comment by luukhoeben** [comment]
I can use any valet command from the C drive, I could also park from the D drive but for example, I cannot restart valet, secure, unsecure, stop, start, all of those things won't work when executed from the D drive, I get that error.

**Comment by cretueusebiu** [comment]
Do you open CMD in C and cd into D or just directly open in D ?

**Comment by luukhoeben** [comment]
I open it in my home directory (so the C drive) and then cd to the D drive, but if I open it in the D drive directly it does the same thing

**Comment by cretueusebiu** [comment]
I've tried few things but the DNS service stops working. You'll have to run all the Valet commands that restart a service from the C drive. So you can run `valet park folder` from the D drive but for example to secure you'll have to run `valet secure some-site` from the C drive.

**Comment by dalevfenton** [comment]
I had this same issue today, trying to secure a site on my D drive. As long as the powershell's current directory is in the C drive the valet secure command works, but if I was in the D drive it didn't. So the workaround is simple, but odd.

**Comment by brainmaniac** [comment]
I also have this problem... I can execute everything from C drive. I have my Code folder on D drive. I successfully parked my Code folder from the D drive.

Running win 10

**Comment by mauroavello** [comment]
@cretueusebiu . First of all thanks for all the work you've put in the project. 
I was going to suggest that if possible to include a note in the home page mentioning this small caveat as until reading this thread I couldn't figure out what I was doing wrong :)

Thanks

**Comment by cretueusebiu** [comment]
@mauroavello Good point! https://github.com/cretueusebiu/valet-windows/commit/f70cb17e24bcb27b228b3b5189c2aaeaa5f416e8

**Comment by zymawy** [comment]
You Need To Config Your  Acrylic DNS
` http://mayakron.altervista.org/wikibase/show.php?id=AcrylicWindows10Configuration`

Not Only For Windows 10 
I'm Using windows 7 
Indeed It's Worked

**Comment by aditya43** [comment]
@zymawy that link seems to be down. Could you please update the link or share what exactly you did to get **_valet secure site-name_** working?

I'm on windows 7 32-bit

**Comment by cretueusebiu** [comment]
@aditya43 `valet secure` should work without manually configuring your network. Valet will try to configure your network automatically when you install it, but if it doesn't you can do it manually ([Win7](http://mayakron.altervista.org/wikibase/show.php?id=AcrylicWindows7Configuration) and [Win10](http://mayakron.altervista.org/wikibase/show.php?id=AcrylicWindows10Configuration))

**Comment by mauroavello** [comment]
Hi @cretueusebiu, I'm having problems once again after I upgraded. I get the previous phpftmservice error, but before that it doesn't seem to be able to start the acrylic service. I think that the previous installation might be interfering and valet uninstall doesn't seem to remove the whole instalation.
Is there any way to remove nginx, acrylic and any other service manually?

Thanks in advance,

Mauro

**Comment by cretueusebiu** [comment]
@mauroavello When you run `valet stop` and `valet uninstall` all services will be removed. Make sure to run these commands in a terminal with administrator access.

**Comment by mauroavello** [comment]
Something is not working....
![acrylic uninstall error](https://user-images.githubusercontent.com/2853299/35238332-6a39f19c-ffad-11e7-8eb4-82e67bae06bb.PNG)
and then after uninstall it's still there
![valet uninstall](https://user-images.githubusercontent.com/2853299/35238362-82667e48-ffad-11e7-8bca-58073c7516db.PNG)

Thanks

**Comment by cretueusebiu** [comment]
That error is thrown by the Acrylic service when you tried to uninstall it but it was never installed. Can't be prevented. 

You can email me and I can take a look via TW.

**Comment by zymawy** [comment]
@mauroavello 
Will I Had This Issue 
What I Did Are 

1 -  Uninstall The VW From Global Composer `composer global remove cretueusebiu/valet-windows`
2 -  Stop All The Services From The Setting In Your Windows 
3 - Delete `C:Users/yourusername/.valet`  Folder 
4 - IMPORTANT ! run The Command Line As Admin
5 -  run `composer global require cretueusebiu/valet-windows`
6 - then `valet install`

This Should Worked

**Comment by mauroavello** [comment]
Good deal @zymawy, that did the trick. Thanks a lot for your help.
I thought that I had removed all the stuff that you mentioned, but didn't work in the order or how I did it.

Thanks again everyone for the time.

**Comment by irvv17** [comment]
if u got error followthe steps of @zymawy works like charm!

dont forget 
4 - IMPORTANT ! run The Command Line As Admin
